CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ set(CMAKE_EXE_LINKER_FLAGS_RELEASE_INIT "-Wl,--gc-sections -Wl,-z,combreloc")
 set(PROJECT_NAME irods_client_rest_cpp)
 string(REPLACE "_" "-" PROJECT_NAME_HYPHENS ${PROJECT_NAME})
 
+set(IRODS_CLIENT_REST_CPP_DIRNAME "${PROJECT_NAME}")
```

Comment:
I separated this from `PROJECT_NAME` in case we change it one day or change our minds again before 1.0.0.